specification/components/schemas/eRS-ReferralRequest.yaml

```diff
@@ -75,6 +75,7 @@ properties:
     type: string
     enum:
       - active
+      - cancelled
       - completed
     example: 'active'
```

Comment:
should the reason and comment be in the referral request now?

ERSSUP-57567 A/C 1.21.1.5

Reply:
@nhsd-david-wass The cancellation reason and comment will be included later once Defining state extension is include and it was agreed with TL and BAs.